gio/Gir.toml

```diff
@@ -822,6 +822,14 @@ status = "generate"
     name = "set_attribute_stringv"
     # use strv
     manual = true
+    [[object.function]]
+    name = "get_attribute_byte_string"
+    # use CString
+    manual = true
+    [[object.function]]
+    name = "set_attribute_byte_string"
+    # use CStr
+    manual = true
```

Comment:
Hm we can't do `string_type = "c_string"`... that only handles all the other options. I wonder how that got forgotten :)